tests/testthat/test-msg.R

```diff
@@ -207,33 +207,47 @@ test_that("error_time_column_must_be_posixct()", {
 })
 
 test_that("error_inconsistent_cols()", {
-  skip("msg")
+  expect_equal(
```

Comment:
This is a regression test so should use `expect_known_output()`

Reply:
I've added an issue so we don't forget. All the tests in this file are currently written like this.